firebase-firestore/src/androidTest/java/com/google/firebase/firestore/QueryTest.java

```diff
@@ -452,6 +452,26 @@ public void testQueriesCanUseInFilters() {
     assertEquals(asList(docF), querySnapshotToValues(snapshot));
   }
 
+  // TODO(in-queries): Re-enable in prod once feature lands in backend.
```

Comment:
You might consider tagging the line below with a bug:

```java
Assume.assumeTrue(isRunningAgainstEmulator());  // TODO(b/xxxxxxxxx)
```

It's not immediately clear that this TODO specifically applies to removing the `assumeTrue`.